doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1039 @@
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.3.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.3.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.3.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_230_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+MCUboot
+=======
+
+* Updated:
+
+  * MCUBoot now uses the Secure RAM region on Trustzone-enabled devices.
+    The option :kconfig:option:`CONFIG_MCUBOOT_USE_ALL_AVAILABLE_RAM` was added to allow MCUBoot to use all the available RAM.
+
+Application development
+=======================
+
+* Added a new user guide about :ref:`app_approtect`.
+
+Build system
+------------
+
+* Removed:
+
+  * Manifest file entry ``mbedtls-nrf`` (:makevar:`ZEPHYR_MBEDTLS_NRF_MODULE_DIR`) checked out at path :file:`mbedtls`.
+  * Manifest file entry ``tfm-mcuboot`` (:makevar:`ZEPHYR_TFM_MCUBOOT_MODULE_DIR`) checked out at path :file:`modules/tee/tfm-mcuboot`.
+
+* Updated:
+
+  * Manifest file entry ``mbedtls`` (:makevar:`ZEPHYR_MBEDTLS_MODULE_DIR`) checked out at path :file:`modules/crypto/mbedtls` now points to |NCS|'s fork of MbedTLS instead of Zephyr's fork.
+
+Working with nRF52 Series
+=========================
+
+* Developing with nRF52 Series:
+
+  * Added Kconfig options :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` and :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` to configure FOTA updates over Bluetooth Low Energy in the default setup.
+    The default setup uses MCUmgr libraries with the Bluetooth transport layer and requires the user to enable MCUboot bootloader.
+    See details in the FOTA updates section of the :ref:`ug_nrf52_developing` guide.
+
+Working with nRF53 Series
+=========================
+
+* Developing with nRF5340 DK:
+
+  * Added Kconfig options :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` and :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` to configure FOTA updates over Bluetooth Low Energy in the default setup.
+    The default setup uses MCUmgr libraries with the Bluetooth transport layer and requires the user to enable MCUboot bootloader.
+    See details in the FOTA updates section of the :ref:`ug_nrf5340` guide.
+
+* Developing with Thingy:53:
+
+  * Added the :kconfig:option:`CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM` Kconfig option to configure USB CDC ACM to be used as logger's backend by default.
```

Comment:
Hi @bjornspock
Do you think we should mention the Thingy:53 related changes in the NCS 2.3 release highlights?

Reply:
@MarekPieta Yes, I think it would be nice to point out that MCUboot and logging over USB CDC is enabled by default. And that the CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU Kconfig option has been added.

Reply:
@bjornspock  @MarekPieta  The highlights )at the beginning of the release notes) have been discussed and sorted out with PMT.
The information you mention - is this related to a particular sample/application? I'm not sure where to put it.

Reply:
@b-gent Allright, then my points are covered. No further comments from my side.